addons/website/__manifest__.py

```diff
@@ -126,6 +126,7 @@
         'data/website_visitor_demo.xml',
     ],
     'application': True,
+    'pre_init_hook': 'pre_init_hook',
```

Comment:
After discussion with PO, let's not do the re-creation of the user upon website install. Given that it cannot be deleted anymore, it is not very useful.